### ISSUE TITLE
Stacked icon support issue

### ISSUE THREAD

**Issue by jmfirth** [andreypopp/react-fa#32]
I've been unable to figure out a way to use stacked icons without resorting to building my own stack wrapper:

``` jsx
<span className='fa-stack fa-lg'>
  <Icon name='circle' stack='2x' inverse={true} />
  <Icon name='user' stack='1x' />
</span>
```

Is there a better way to accomplish this?  

If not, would you be interested in a PR for adding a new component to encapsulate this?  Perhaps:

``` jsx
<IconStack size='lg'>
  {/* children */}
</IconStack>
```

**Comment by andreypopp** [comment]
👍

**Comment by andreypopp** [comment]
Thanls @jmfirth, it's been released as 4.1.0.

**Comment by jmfirth** [comment]
Thanks for the quick turn-around @andreypopp!
